eland/etl.py

```diff
@@ -22,7 +22,9 @@
 import pandas as pd  # type: ignore
 from elasticsearch import Elasticsearch
 from elasticsearch.helpers import parallel_bulk
+from pandas.core.config_init import is_terminal  # type: ignore
```

Comment:
I'm not sure if `is_terminal` detects only Jupyter Notebook, from the function itself it looks like it mentions IPython which can be used outside of a Jupyter Notebook: https://github.com/pandas-dev/pandas/blob/master/pandas/core/config_init.py#L323

Reply:
Should I do this way?
```python
        try:
            get_ipython()
        except NameError:
            pass
```
`get_ipython()` works in jupyter notebook whereas it fails in terminal.